packages/utilities/psammead-locales/moment/sw.js

```diff
@@ -5,4 +5,37 @@ moment.updateLocale('sw', {
   months: 'Januari_Februari_Machi_Aprili_Mei_Juni_Julai_Agosti_Septemba_Oktoba_Novemba_Disemba'.split(
     '_'
   ),
+  relativeTime: {
```

Comment:
Curious about this approach. It is my understanding that we avoid making major changes to moment, so that these changes can be adopted by moment. Is there merit to having it this way as opposed to:

```
relativeTime: {
    past: 'hivi punde zilizopita',
    m: 'Dakika 1'
    mm: 'Dakika %d zilizopita,
    h: 'Saa 1 iliyopita',
    hh: 'Saa %d zilizopita',
  },
```

Reply:
So that suggestion didn't work, because that essentially sets the value to `hivi punde zilizopita` for any relative time in the past, rather than specifying a value for how long ago it was.

There didn't seem to be a way to put logic inside the `past` translation based on the value of the variable - so to get different words for `iliyopita` and `zilizopita` based on the number, the logic had to live where I've put it.

Reply:
Ok. Thanks for the explanation